libs/components/core/src/lib/modules/numeric/numeric.pipe.spec.ts

```diff
@@ -76,6 +76,22 @@ describe('Numeric pipe', () => {
     }).toThrowError();
   });
 
+  it('should properly handle undefined being passed in', async () => {
```

Comment:
The word "properly" is extraneous. You could be more concise with: `should handle undefined and null values`.

Reply:
Done